t/pcapCoverageScript.t

```diff
@@ -66,13 +80,13 @@ subtest 'gff3 with output file' => sub {
   my ($stdout, $stderr, $exit) = capture {
     system( $cmd );
   };
-  ok($exit==0,"Zero exit code for script");
+  is($exit, 0,"Zero exit code for script");
   my $FH;
   ok(open($FH, '<', $test_output),"Read from test output file");
```

Comment:
Not a major issue, you should just die on standard perl operations in tests, you probably want the code to fall over rather than run all the tests if you can read or write.